packages/instant/src/containers/latest_error.tsx

```diff
@@ -23,7 +24,12 @@ export interface LatestErrorComponentProps {
 
 export const LatestErrorComponent: React.StatelessComponent<LatestErrorComponentProps> = props => {
     if (!props.latestErrorMessage) {
-        return <div />;
+        // Render a hidden SlidingError such that instant does not move when a real error is rendered.
+        return (
+            <Container isHidden={true}>
```

Comment:
Any reason for this approach as opposed to just setting a constant height on the container like you did for `payment_method.tsx`?

Reply:
Seems more flexible... you can now freely change the SlidingError component and not break this logic. 

The reason I used a fixed height `payment_method` is because the height of the children is slightly different, so I wanted to fix the height of the parent which was ultimately the thing making instant bounce around. 

I guess you could argue for the same thing here, but this seemed simpler (I don't like fixed heights)